### PR TITLE
Logans branch

### PR DESCRIPTION
**Pull request by Dolechek**
Please let Devin look over this before accepting the pull request.

### CODE REVIEW

File: topics/books.html [hunk 1]
```diff
@@ -14,10 +14,20 @@ <h1>Methods 2022 - Topics - Books</h1>
...
       <li>Greenlights - Matthew McConaughey</li>
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+      <li>Dune - Frank Herbert</li>
+=======
+      <li>Harry Potter and the Sorcerer's Stone - J.K. Rowling</li>
+	  <li>The Hobbit - J.R.R Tolkien</li>
+>>>>>>> d68c36b59e3881d943eed9aeb11e7782e65921f1
+=======
```
Comment (sleddog) [inline topics/books.html:25]: clean up these lines... this is showing a merge conflict

File: topics/cookies.html [hunk 2]
```diff
@@ -12,8 +12,13 @@ <h1>Methods 2022 - Topics - Favorite Kind of Cookie</h1>
       <li>Classic chocolate chip</li>
       <li>Peanut butter cholocate chip</li>
       <li>Nuttell stuffed cookies</li>
+<<<<<<< HEAD
+      <li>GingerSnaps</li>
+      <li>Peanut Butter Cookies</li>
+=======
       <li>Frosted sugar cookie</li>
 	  <li>White chocolate chip macodamia nut</li>
+>>>>>>> d68c36b59e3881d943eed9aeb11e7782e65921f1
```
Comment (sleddog) [inline topics/cookies.html:21]: same here

File: topics/food.html [hunk 3]
```diff
@@ -9,10 +9,17 @@ <h1>Methods 2022 - Topics - Favorite Food</h1>
...
+<<<<<<< HEAD
+		<li>Mashed Potatoes</li>
+=======
+=======
 		<li>Jajangmyeon</li>
+>>>>>>> 9e0acf977799ed9e6dbd9cf0276600867d79ec87
 		<li>Tacos and fajitas</li>
 		<li>Croissant</li>
 		<li>Chicken alfredo</li>
+>>>>>>> d68c36b59e3881d943eed9aeb11e7782e65921f1
```
Comment (sleddog) [inline topics/food.html:22]: here

File: topics/index.html [hunk 4]
```diff
@@ -19,7 +19,11 @@ <h1>Methods 2022 - Topics</h1>
       <li><a href="./colors.html">Colors</a></li>
       <li><a href="./animals.html">Animals</a></li>
       <li><a href="./plants.html">Plants</a></li>
+<<<<<<< HEAD
+      <li><a href="./hunting.html">Hunting</a></li>
+=======
       <li><a href="./activities.html">Activities</a></li>
+>>>>>>> 9e0acf977799ed9e6dbd9cf0276600867d79ec87
```
Comment (sleddog) [inline topics/index.html:26]: here

File: fizzbuzz/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+This is a simple instructional guide how to run my python code with a shell script. 
+
+## Directions
+
+1. Make sure you're in fizzbuzz/python/loganDolechek
+2. In your command line type ./sb.sh # 
+3. # being the number you want to input to the python code. 
+4. Press enter. 
+5. You should be prompted to input a number of your choosing. Do so & press enter.
```
Comment (Sawahh) [inline fizzbuzz/README.md:11]: Could you adjust your code to only ask for the number to be used in your fizzbuzz once?

**Comment by Sawahh** [comment]
Attempting to do a +10 I got this error while following the ReadMe
image

**Pull request merged**